### PR TITLE
Reviewed and reworked the policy based on the input from AnBuKu/hugi

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

File: docs/code-signing-policy.rst [hunk 1]
```diff
@@ -27,43 +37,31 @@ DebOps Developer.
...
 their account on the source code management platform used to work on the DebOps
-Project (currently GitHub) it RECOMMENDED to provide a proof via the means of
+Project (currently GitHub) it is RECOMMENDED to provide a proof via the means of
 https://keybase.io/.
 
 Additionally, it is RECOMMENDED to take part in the Web Of Trust to make
 it harder for an adversary to fake signatures by pretending to be one of the
 DebOps Contributors or Developers. In particular as the DebOps Project is related to the Debian
-Project it is RECOMMENDED to get your key signed by Debian Developers.
+Project it is RECOMMENDED to get your key signed by at least one Debian Developers.
```
Comment (drybjed) [inline docs/code-signing-policy.rst]: In this case this probably should be 'at least one Debian Developer'.

File: docs/index.rst [hunk 2]
```diff
@@ -11,6 +11,7 @@ DebOps Policies and Guidelines
    distributed-development
    code-signing-policy
    security-policy
+   planed-policy-changes
```
Comment (drybjed) [inline docs/index.rst]: Shouldn't this be 'planned-policy-changes'? https://oilpatchwriting.wordpress.com/2010/08/12/planed-vs-planned/

File: docs/organizational-structure.rst [hunk 3]
```diff
@@ -16,6 +16,10 @@ The Project is managed and developed by a decentralized group of people who
 use it and contribute their time, skills and resources. To allow for a more
 organized work, various roles are defined within the Project.
 
+In the industry the term "organizational structure" might also be know as
```
Comment (drybjed) [inline docs/organizational-structure.rst]: "known"

**Comment by ypid** [comment]
@drybjed Thanks. Fixed.

**Pull request merged**